lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -130,10 +141,46 @@ private NettyMessagingTransport(
 
     final String host = UNKNOWN_HOST_NAME.equals(hostAddress) ? localAddressProvider.getLocalAddress() : hostAddress;
 
+    final SslContext sslContextClient;
+    final SslContext sslContextServer;
+
+    ssl = System.getProperty("ssl") != null;
```

Comment:
It would be good to provide an option for use whether use SSL or not. I would redesign the implementation after refactoring the Wake Transport. Thank you!